inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/function/EncryptFunction.java

```diff
@@ -0,0 +1,190 @@
+     * @return
+     */
+    public static String encrypt3DES(String data, String key) {
+        try {
+            SecretKey desKey = new SecretKeySpec(build3DesKey(key),
+                    KEY_ALGORITHM_3DES);
+            Cipher cipher = Cipher.getInstance(CIPHER_ALGORITHM_3DES);
+            cipher.init(Cipher.ENCRYPT_MODE, desKey);
+            return encryptBASE64(cipher.doFinal(data.getBytes(ENCODING)));
+        } catch (Exception e) {
+            return null;
+        }
+    }
+
+    /**
+     * build 3DesKey
+     * @param keyStr the key of encryption
+     * @return
+     */
+    public static byte[] build3DesKey(String keyStr) {
+        try {
+            byte[] key = "000000000000000000000000".getBytes(ENCODING);
+            byte[] temp = keyStr.getBytes(ENCODING);
+            if (key.length > temp.length) {
+                System.arraycopy(temp, 0, key, 0, temp.length);
+            } else {
+                System.arraycopy(temp, 0, key, 0, key.length);
+            }
+            return key;
+        } catch (Exception e) {
+            return null;
+        }
+    }
+
+    /**
+     * encrypt by BASE64
+     *
+     * @param plaintextBytes it is data to be encoded
+     * @return
+     */
+    public static String encryptBASE64(byte[] plaintextBytes) throws Exception {
+        return new String(base64.encode(plaintextBytes), ENCODING);
+    }
+
+    /**
+     * encrypt by AES
+     *
+     * @param content it is content to be encrypted
+     * @param key the key of encryption
+     * @return
+     */
+    public static String encryptAES(String content, String key) {
+        try {
+            KeyGenerator kgen = KeyGenerator.getInstance(KEY_ALGORITHM_AES);
+            SecureRandom random = SecureRandom.getInstance(SIGN_ALGORITHMS);
+            random.setSeed(key.getBytes(ENCODING));
+            kgen.init(128, random);
+            SecretKey secretKey = kgen.generateKey();
+            byte[] enCodeFormat = secretKey.getEncoded();
+            SecretKeySpec secretKeySpec = new SecretKeySpec(enCodeFormat, KEY_ALGORITHM_AES);
+            Cipher cipher = Cipher.getInstance(KEY_ALGORITHM_AES);
+            byte[] byteContent = content.getBytes(ENCODING);
+            cipher.init(Cipher.ENCRYPT_MODE, secretKeySpec);
+            byte[] byteRresult = cipher.doFinal(byteContent);
+            StringBuffer sb = new StringBuffer();
+            for (int i = 0; i < byteRresult.length; i++) {
+                String hex = Integer.toHexString(byteRresult[i] & 0xFF);
+                if (hex.length() == 1) {
+                    hex = '0' + hex;
+                }
+                sb.append(hex.toUpperCase());
+            }
+            return sb.toString();
+        } catch (Exception e) {
+            e.toString();
+        }
+        return null;
+    }
+
+    /**
+     * Encryption Algorithm Enum
+     */
+    private enum EncryptionAlgorithmEnum {
+
+        /**
+         * DESEDE
+         */
+        DESEDE(1),
```

Comment:
It is recommended to add a getInstance method to get the enumeration object, if not get the corresponding exception thrown.